com.unity.robotics.urdf-importer/Runtime/MeshProcessing/StlAssetPostProcessor.cs

```diff
@@ -26,61 +26,69 @@ public class StlAssetPostProcessor : AssetPostprocessor
     public class StlAssetPostProcessor
 #endif    
     {
+        private static Material s_DefaultDiffuse = null;
+        
         static void OnPostprocessAllAssets(string[] importedAssets, string[] deletedAssets, string[] movedAssets, string[] movedFromPath)
         {
+            const bool enableStlPostprocessing = false; // AIRO-908 
```

Comment:
Unnecessary bool initialization.

Reply:
removed the entire function. please see above.